Update transaction-life-cycle.adoc (#1032)
starknet-io/starknet-docs: components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/transaction-life-cycle.adoc

|`SUCCEEDED` |The transaction was successfully executed by the sequencer. It is included in the block.
|===

> Note that if the finality status of a transaction is `RECEIVED` or `REJECTED`, then it will not have an execution status.

Reviewer: ```suggestion
[NOTE]
====
If the finality status of a transaction is `RECEIVED` or `REJECTED`, the transaction does not have an execution status.
====
```
